scripts/generic_tests.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright © 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+cmd_help() {
+    echo ""
+    echo "Cloud Hypervisor $(basename $0)"
+    echo "Usage: $(basename $0) <command> [<command args>]"
+    echo ""
+    echo "Available commands:"
+    echo ""
+    echo "    run --ch --kernel --cmdline --rootfs --cloudinit --guest-user --guest-passwd --mac --ip --mask --guest-ip [--device] --script"
```

Comment:
Out of all those options, which ones are mandatory and which ones are not?

Reply:
Providing we can generate a default cloud-init image, the mandatory options are: `ch`, `kernel`, `rootfs`